packages/dd-trace/src/profiling/profilers/events.js

```diff
@@ -339,35 +342,12 @@ class CompositeEventSource {
 }
 
 function createPossionProcessSamplingFilter (samplingIntervalMillis) {
```

Comment:
nit: 
```suggestion
function createPoissonProcessSamplingFilter (samplingIntervalMillis) {
```

Reply:
My fingers' muscle memory will apparently never learn this, [exhibit A](https://github.com/DataDog/dd-trace-js/pull/6089/files#diff-7b72eb3a4fd3dbce3d567ab882ca75d313c047a6ff49e202adb4fc71f470c397L341-R369).